mod/profile.php

```diff
@@ -6,78 +6,81 @@
 use Friendica\Core\System;
 use Friendica\Database\DBM;
 
-require_once('include/contact_widgets.php');
-require_once('include/redir.php');
+require_once 'include/contact_widgets.php';
+require_once 'include/redir.php';
 
-function profile_init(App $a) {
-
-	if(! x($a->page,'aside'))
+function profile_init(App $a)
+{
+	if (!x($a->page, 'aside')) {
 		$a->page['aside'] = '';
+	}
 
-	if($a->argc > 1)
+	if ($a->argc > 1) {
 		$which = htmlspecialchars($a->argv[1]);
-	else {
-		$r = q("select nickname from user where blocked = 0 and account_expired = 0 and account_removed = 0 and verified = 1 order by rand() limit 1");
+	} else {
+		$r = q("SELECT `nickname` FROM `user` WHERE `blocked` = 0 AND `account_expired` = 0 AND `account_removed` = 0 AND `verified` = 1 ORDER BY RAND() LIMIT 1");
 		if (DBM::is_result($r)) {
 			goaway(System::baseUrl() . '/profile/' . $r[0]['nickname']);
-		}
-		else {
+		} else {
 			logger('profile error: mod_profile ' . $a->query_string, LOGGER_DEBUG);
-			notice( t('Requested profile is not available.') . EOL );
+			notice(t('Requested profile is not available.') . EOL);
 			$a->error = 404;
 			return;
 		}
 	}
 
 	$profile = 0;
-	if((local_user()) && ($a->argc > 2) && ($a->argv[2] === 'view')) {
+	if (local_user() && $a->argc > 2 && $a->argv[2] === 'view') {
```

Comment:
I - personally - prefer this line in the previous state.like I'm all in for removing the brackets for single values. But here they increase readability.

Reply:
I believe parentheses should be used when they are necessary because they increase clutter otherwise. It's particularly true when function calls (with parentheses) are wrapped in parentheses.

Reply:
I like to use them not only when they are needed but also when they help to understand the condition. I need more time to understand a complex condition without any parentheses (especially because at work I'm using a language that needs them)